workflow/rules/utils.smk

```diff
@@ -66,16 +69,13 @@ def _get_biosample_model_dir(biosample):
 	else:
 		return os.path.join(MODEL_DIR, f"{access_type}_intact_hic")
 
-def get_feature_table_file(biosample):
-	model_dir = _get_biosample_model_dir(biosample)
+def get_feature_table_file(model_dir):
```

Comment:
Did these functions need to be changed? The behavior of these are so simple, they don't even need to be functions. See one of my comments above on how to utilize these functions